duckduckgo/Android
# Release Fireproof login detection

Task/Issue URL: https://app.asana.com/0/488551667048375/1200078998119893/f
Tech Design URL: 
CC: https://app.asana.com/0/1142021229838617/1200081522320196/f

**Description**:
We decided to release to all users Fireproof login detection enabled by default (https://github.com/duckduckgo/Android/pull/1097).
This PR removes experimental variants of our login detection experiment.

Changes:
- login detection enabled by default to all users
- removed variants
- copy translations

**Steps to test this PR**:
1. Fresh install
1. Open the browser
2. Go to settings, ensure login detection is enabled by default

**Optional test cases (we haven't introduced any change in our login detection logic, I'm copying here the most important ones from https://github.com/duckduckgo/Android/pull/1097):**

Test 1: when Login dialog dismissed twice in a row, ask user to disable login detection
1. Fresh intall
2. Go to settings and ensure login detection is **enabled**
3. Go back to the browser
4. Perform login in **two** different websites, and dismiss the Fireproof login dialog in all of them
5. Ensure dialog asking to disable login detection shown
6. Press disable
7. Go to fireproof settings and ensure login detection is disabled

Experiment Test 2: we ask to disable login detection only once
1. Fresh intall
4. Perform login in **two** different websites, and dismiss the Fireproof login dialog in all of them
5. Ensure dialog asking to disable login detection shown
6. Press "Cancel"
7. Clear all data (this is optional, clear data so we can login on sites again)
4. Perform login in **two** different websites, and dismiss the Fireproof login dialog in all of them
5. Ensure we don't show again the dialog to disable login detection

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
